package.json

```
@@ -72,6 +72,6 @@
     "typescript": "4.6.3"
   },
   "engines": {
-    "node": ">=16 <=18"
+    "node": ">=16 <=20"
```

Now that Node 16 is EOL?
```suggestion
    "node": ">=18 <=20"
```